### ISSUE TITLE
Does not work without Python 3

### ISSUE THREAD

**Issue by Gaibhne** [garabik/grc#116]
Despite the README claiming otherwise, this project does not run without Python 3. The following error message is displayed when `grc` is called on a system with only Python 2 installed:

`/usr/bin/env: python3: No such file or directory`

**Comment by garabik** [comment]
Yes, you are supposed to change the first line of /usr/(local/)/bin/grc and /usr/(local/)/bin/grcat from python3 to python. Unfortunately, it is kind of difficult to do automatically, but at least I'll update the README. And, given current year, I'd rather support python3 primarily (though python3 is better suitable for this, because of the lack of automagic unicode decoding).

**Comment by briantully** [comment]
Aha!!! I was going crazy with this issue, and removing grc was the only way around it. Glad to hear there is a workaround. I don't see this workaround in the the README file though :(
